src/app/userDashboard/store/reducers.ts

```diff
@@ -0,0 +1,60 @@
+import {
+  ActionReducer,
+  ActionReducerMap,
+  INIT,
+  MetaReducer,
+  createReducer,
+  on,
+} from '@ngrx/store';
+import { TripState } from '../../models/state';
+import { getTripsComplete } from './actions';
+import { isDevMode } from '@angular/core';
+import {
+  logoutUser,
+  logoutUserComplete,
+} from '../../userManagement/store/actions';
+
+export const userDashboardFeatureKey = 'userDashboardFeature';
+
+const intialState: TripState = {
+  trips: undefined,
+};
+
+export const userDashboardReducers = createReducer(
+  intialState,
+  on(getTripsComplete, (state, { trips }) => ({
+    ...state,
+    trips,
+  })),
+  on(logoutUserComplete, (state, {}) => ({
+    ...state,
+    trips: undefined,
+  }))
+);
+
+export const reducers: ActionReducerMap<TripState> = {};
+
+const debugMeta = (
+  reducer: ActionReducer<TripState>
+): ActionReducer<TripState> => {
+  return (state, action) => {
+    console.log('state', state);
```

Comment:
whoops, console.logs

Reply:
This is part of the meta reducer I need it for development to log the actions